app/services/weets.js

```diff
@@ -46,8 +46,36 @@ const readWeets = async ({ page = 1, limit = 5 }) => {
   }
 };
 
+const getWeet = async weetId => {
+  try {
+    const weet = await db.Weet.findOne({ where: { id: weetId } });
+    if (!weet) {
+      throw errors.notFoundError('Could not find the weet requested');
+    }
+    return weet;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getUserWeets = async userId => {
+  try {
+    const userWeets = await db.Weet.findAll({
+      where: { userId }
+    }).map(weet => weet.dataValues.id);
```

Comment:
Here you should return the whole weets in the arrays and extract the ids where you use this function